src/index.js

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
```

Comment:
Kindly remove this line of code (`import report....') especially since it is of no use in the application. This will make your code follow the YAGNI principle. 👍